tutorials/assets/fdem/fdem_data.obs

```diff
@@ -0,0 +1,243 @@
+1.0000e+02 -2.0000e+02 -2.0000e+02 2.0000e+01 -4.4874e-14 -3.3585e-13
```

Comment:
instead of including these in the repo, would it work if we archived them on google and just download as needed? I am happy to help with that if that works with folks